docs/getting_started/installation_guide/platforms/custom.rst

```diff
@@ -186,53 +182,55 @@ The following components are supported by TLS version 1.2+:
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| ndbmtd            | mysql      | mysql     | 10000                | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| mysqld            | mysql      | mysql     | 3306                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| ndb_mgmd          | mysql      | mysql     | 1186                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| hiveserver2       | hive       | hive      | 9084-9085            | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| metastore         | hive       | hive      | 9083                 | Yes if external access to the Feature Store is needed          |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| kafka             | kafka      | kafka     | 9091, 9092(external) | Yes for 9092 if external access to the Kafka cluster is needed |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| zookeeper         | zookeeper  | zookeeper | 2181                 | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| epipe             | epipe      | epipe     | 29191 (metrics)      | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| airflow-scheduler | airflow    | airflow   |        N/A           | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| sqoop             | airflow    | airflow   | 16000                | No                                                             |
++-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
+| airflow-webserver | airflow    | airflow   | 12358                | No                                                             |
```

Comment:
Prometheus and Grafana are notable absents to the list. Both of them are interesting if you want to plug your cluster in another monitoring service.

Reply:
Sparkhistoryserver is also missing. K8s is also missing.

Reply:
fixed